### ISSUE TITLE
Blurry Images

### ISSUE THREAD

**Issue by AbigaelElenara** [imazen/resizer#244]
TL;DR: At some unspecified time after updating to Anki 2.1 and decreasing Anki's webscale, any images I resize - and previously resized images while I'm reviewing - are blurry. Normal images seem fine, except when I use Snipping Tool I notice a bit more blur.
I've tried all sorts of settings with Image Resizer, restarted Anki, edited the .bat file to 1.00... nothing.

I have been using Image Resizer successfully for a long time. Recently I updated to Anki 2.1 and had another issue where everything was too zoomed in, so I had to make a .bat file and make the web scale smaller. Yesterday I noticed these blurry images.
When I just paste images in, they look okay; I can't tell much difference in the resolution. But when I resize them, they turn blurry, which is worrying because I use Anki all day for homeschool and I can't leave the images full-size because of that over-zooming thing - they'd be even bigger than normal. Example without and with resizing:
<img width="775" alt="anki2" src="https://user-images.githubusercontent.com/33943723/55179268-16512f00-5155-11e9-9397-e7dd33b1ab61.png">
<img width="813" alt="anki" src="https://user-images.githubusercontent.com/33943723/55179275-18b38900-5155-11e9-9ed4-81632322e46c.png">
It's currently set to resize by height to 200 pixels, but I'd had it at 400 and 300 before with the same issues.
I just tried to edit the .bat file and reset the webscale back to 1.00 and see if that was the issue, but it's still doing the same thing!

Edit: I'm pretty sure this is happening even with files I resized a long time ago. I'm 99% sure this picture was crystal clear before (it looks veeery blurry in the full-sized window):
<img width="908" alt="file" src="https://user-images.githubusercontent.com/33943723/55179951-8f04bb00-5156-11e9-923b-b8e52a365bf2.png">

More experimentation: This MIGHT be a problem with images in general, just intensified by Image Resizer for some reason. I didn't resize this image, and it's definitely blurrier in Anki. That said, this is a screenshot, not a copy-pasted image - when I copy an image directly as I did above, I only notice the blurriness with IR.
<img width="427" alt="anki3" src="https://user-images.githubusercontent.com/33943723/55180701-1ef73480-5158-11e9-975e-018de6c54fe3.png">
<img width="427" alt="nada" src="https://user-images.githubusercontent.com/33943723/55180712-24547f00-5158-11e9-9cac-525aa0e513f6.png">
And the worst of all is when I do resize it:
<img width="383" alt="anki4" src="https://user-images.githubusercontent.com/33943723/55180754-39c9a900-5158-11e9-9b80-1d61d2344ce5.png">

**Comment by AbigaelElenara** [comment]
Bump?
I'd love to get help with this because I use Anki constantly & every day means more blurry images possibly-permanently added to my decks!

**Comment by lilith** [comment]
I think you have the wrong repository; this is for a web server module.

**Comment by AbigaelElenara** [comment]
Ohh, thank you... I was trying to open one for an Anki extension called Image Resizer and... either the extension must use this image-resizing program and it took me here when I tried to find where to leave a comment on the extension, or I just looked up the wrong thing when muddling around trying to figure out how to use GitHub.
Alright I found the actual place I need to be! Sorry about that, thanks for telling me.
